src/main/java/salesforce/classic/WorkTypeForm.java

```diff
@@ -22,13 +21,13 @@ public WorkTypeForm(final WebDriver driver) {
 
     @Override
     protected void waitForPageLoaded() {
-        wait.until(ExpectedConditions.visibilityOf(nameTxtBox));
+        webElementAction.waitForElementVisibility(nameTxtBox);
     }
 
     /**
      * Sets the name of work type.
      *
-     * @param workTypeName
+     * @param workTypeName is name work type
      */
     public void setNameTxtBox(final String workTypeName) {
         nameTxtBox.sendKeys(workTypeName);
```

Comment:
I think you could use the web actions you already define so you don't have to repeat the same process

Reply:
ok a forgot that